packages/marshal/src/types.js

```diff
@@ -180,7 +182,7 @@
 
 /**
  * @template Slot
- * @typedef CapData
+ * @typedef {Object} CapData
```

Comment:
Good catch!

Reply:
Curious why `tsc` doesn't report this. Is there a valid interpretation? I suppose it's more lenient with JSDoc than code syntax.

Reply:
Confirmed, the earlier expression worked the same.